gavel/templates/admin.html

```diff
@@ -1,6 +1,66 @@
-{% extends "layout.html" %}
-{% block title %}Admin{% endblock %}
-{% block content %}
+{% extends "layout.html" %} {% block title %}Admin{% endblock %} {% block
+content %}
+
+<div class="item">
+  <div class="banner">
+    <h1>Welcome</h1>
+  </div>
+  <div class="info">
+    {{ content }}
+    <div class="info__header">
+      <h1>Hey Javier,</h1>
+    </div>
+
+    <div class="info__guidance">
+      <div class="info__guidance--heading">
+        <h3>How to use Gavel</h3>
+      </div>
+
+      <div class="info__guidance--steps">
+        <div>
+          <h4>First step</h4>
+          <p>
+            You will see one project only to review, when you are done the
+            comparison starts
+          </p>
+        </div>
+        <div>
+          <h4>Compare projects</h4>
+          <p>
+            You will be comparing two projects at a time, from which you have to
+            select one as better than the other
+          </p>
+        </div>
+        <div>
+          <h4>Skipping projects</h4>
+          <p>
+            If a project seems inappropriate, violates event rules or it is not
+            working properly, you can choose the option to skip it
+          </p>
+        </div>
+      </div>
+
+      <div class="info__guidance--footer">
+        <p>
+          If you have problems reviewing the projects, please contact the event
+          organizer
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="info">
+    <form action="{{ url_for('welcome_done') }}" method="post">
+      <input
+        type="submit"
+        name="action"
+        value="Start reviewing"
+        class="button-full button-gap positive"
+      />
+      <input type="hidden" name="_csrf_token" value="{{ csrf_token() }}" />
+    </form>
+  </div>
+</div>
+
```

Comment:
The styling is on point, and I think you used it here to test the visuals, but forgot to remove it from the admin view